cmd/thanos/compact.go

```diff
@@ -122,6 +122,9 @@ func registerCompact(m map[string]setupFunc, app *kingpin.Application) {
 	compactionConcurrency := cmd.Flag("compact.concurrency", "Number of goroutines to use when compacting groups.").
 		Default("1").Int()
 
+	deleteDelay := modelDuration(cmd.Flag("delete-delay", fmt.Sprintf("Time before a block marked for deletion is deleted from bucket")).
+		Default("15m"))
```

Comment:
hm.. Right now out threshold was `	PartialUploadThresholdAge = 2 * 24 * time.Hour` I would start with just delay instead maybe?

Reply:
are you saying that the default value for deleteDelay should be 48 hours?